### PR TITLE
ISLANDORA-2363: ffmpeg safe pixel format specification

### PR DESCRIPTION
**Pull request by bradspry**
By default, the solution pack generates MP4s which only playback successfully in Chrome. To create a "safe" MP4 which plays back in all browsers, ffmpeg **[recommends](https://trac.ffmpeg.org/wiki/Encode/H.264#QuickTime)** the following switch:

-vf format=yuv420p

https://jira.duraspace.org/browse/ISLANDORA-2363

### CODE REVIEW

**Comment by DonRichards** [comment]
Just wondering why apply `-vf format=yuv420p`? According to that recommendation it specified to that for quicktime but under the **Compatibility > All devices** that doc suggested to use `-profile:v baseline -level 3.0`. Just curious.

**Comment by bradspry** [comment]
@DonRichards, after further research and testing, I can definitively say "-profile:v baseline -level 3.0" also achieves YUV420p, resulting in a video that's playable in all browsers (all the ones I have for testing).

In summary, either choice you make, the playback goal will be achieved.  

Making no choice however and providing ffmpeg with no instruction, produces a "yuv422p" file that might only work in Chrome.

**Comment by bradspry** [comment]
@DonRichards, today an archivist ingested a yuv422 pixel format video.  Derivatives.inc was set to use "-profile:v baseline -level 3.0" 

FFmpeg threw error:
x264 [error]: baseline profile doesn't support 4:2:2
[libx264 @ 0x227df60] Error setting profile baseline.
[libx264 @ 0x227df60] Possible profiles: baseline main high high10 high422 high444

I then updated derivatives.inc to use "-vf format=yuv420p" instead.  Success!

In summary, by using the baseline profile setting, you provide FFmpeg wiggle room to make decisions on its own.  If it runs into a conflict, it effectively bombs and creates a zero length MP4.

By using the yuv420p setting, you afford FFmpeg no wiggle room and no fuzziness.  By providing a clear instruction to the encoder, success is found.

**Comment by DonRichards** [comment]
Good stuff, thanks for that.

**Comment by jonathangreen** [comment]
@Islandora/7-x-1-x-committers any committers have any feedback on this flag change? I'm going to do some testing this week and try to get this merged if no one has any feedback to the contrary.

**Review by jonathangreen — APPROVED** [review]
Tested and it's working for me. 

Starting merge ⏰
